### ISSUE TITLE
Add ghost image based on node v10.15.0

### ISSUE THREAD

**Issue by aparaschiv** [docker-library/ghost#187]
Currently ghost image is generated using node 10.x latest.
It would be nice to have an official image based on node 10.15.0 since it doesn't have the ENOTCONN [issue]( https://github.com/nodejs/node/issues/26315) when using tls connections with mysql, and I don't think it will get fixed very soon.

**Comment by wglambert** [comment]
The current one we're using is v10.16

```console
$ docker run --rm ghost:latest node -v
Unable to find image 'ghost:latest' locally
latest: Pulling from library/ghost
fc7181108d40: Pull complete 
a108b1bb7629: Pull complete 
c7ba6ff99c0f: Pull complete 
5776cc831b44: Pull complete 
b2005dcb3156: Pull complete 
e9aa34e83ebb: Pull complete 
efff56921755: Pull complete 
73b7aa064442: Pull complete 
cd95522ec6f7: Pull complete 
Digest: sha256:df6f0d0b896b733939c456323f9473c8db5fac8d5822dd565ea428f9c4ea5237
Status: Downloaded newer image for ghost:latest
v10.16.0
```

It looks like Node only maintains the latest minor versions in every supported major version, so this isn't really something we would want to maintain.

To get the Node 10.15.0 Dockerfile you'd have to grab an [archived copy](https://github.com/docker-library/repo-info/blob/a9a90625361d629317650329b8c5fe9632f54c67/repos/node/remote/10.15.0-stretch-slim.md) and then add ghost on top of that using our Dockerfile as a reference/template

The earliest 10.15 version we've ever used is 15.3 with https://github.com/docker-library/repo-info/blob/e1d4df3fcad108e481c24a2d59da430da0f9f07e/repos/ghost/remote/2.19.1.md
From https://github.com/docker-library/official-images/pull/5615 -> https://github.com/docker-library/ghost/pull/179, so the latest Node version when we switched to v10 on March 25 was 10.15.3

https://github.com/docker-library/repo-info/blob/e1d4df3fcad108e481c24a2d59da430da0f9f07e/repos/ghost/remote/2.19.1.md
```console
$ docker run --rm ghost@sha256:fba10f78762a639c8431d37af7f831266ade75f253effc9d6c519a6374d9a457 node -v
v10.15.3
```

**Comment by aparaschiv** [comment]
In that case, we can close this request as it adds unnecessary complexity. 
As a workaround until the main issue is fixed I'm using `FROM node:10.15.0-alpine` with latest version of Ghost.
Haven't encountered any issues so far.
Thanks!

**Comment by wglambert** [comment]
I forgot you could get old versions without the sha256 hash

So making your own Dockerfile with that `FROM node:10.15.0` line is simpler
